Dart/resources/liveTemplates/dart_miscellaneous.xml

```diff
@@ -45,5 +45,21 @@
       <option name="DART" value="true"/>
     </context>
   </template>
+
+  <template name="sout" value="print('$END$');" description="print to standard out"
+            toReformat="true" toShortenFQNames="true">
+    <context>
+      <option name="DART" value="true"/>
+    </context>
+  </template>
+
+  <template name="soutm" value="print('$CLASS_NAME$.$METHOD_NAME$');&#10;$END$" description="prints the current class and method name to standard out"
```

Comment:
Line break at the end is unexpected to me. As I see no other similar templates have it. Was it intentional?